src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -1135,6 +1135,9 @@ public class CaseData extends Et1CaseData {
     private String tseResponseCopyNoGiveDetails;
     @JsonProperty("resTseTableMarkUp")
     private String resTseTableMarkUp;
+    //  if Respondent is responding to Tribunal
+    @JsonProperty("tseRespondToTribunal")
```

Comment:
Maybe change it to tseRespondingToTribunal

Reply:
Any reason why?

Reply:
I think it's a better name, it's even what you put in the comment. It's not a deal breaker for me but I think it'd be nicer

Reply:
updated